client/src/pages/Map.js

```diff
@@ -83,17 +83,22 @@ export default function Map() {
 
   return (
     <>
-      <SubmitForm handleOnSubmit={handleOnSubmit} />
+      <SubmitForm
+        className={"map__submitform"}
```

Comment:
when the block is your component, you normally write it capitalized like your component.